enhancements/console/dynamic-plugins.md

```diff
@@ -147,6 +147,48 @@ spec:
     basePath: '/'
 ```
 
+In case the plugin needs to communicate with other in-cluster service, it can
+declare a service proxy in it's `ConsolePlugin` resource using the
+`spec.proxy.services` array field. A service `name`, `namespace` and `port`
+needs to specified.
+
+Console backend exposes following endpoint in order to proxy the communication
+between plugin and the service:
+`/api/proxy/namespace/<service-namespace>/service/<service-name>:<port-number>/<request-path>?<optional-query-parameters>`
+
+An example proxy request path from plugin to `helm-charts` service,
+in `helm` namespace to list ten helm releases:
+`/api/proxy/namespace/helm/service/helm-charts:8443/releases?limit=10`
+
+Proxied request will be using using the service CA bundle by default. For
```

Comment:
```suggestion
Proxied request will use the service CA bundle by default. For
```

Reply:
I think we should mention here something like "default Console service CA bundle" or similar, not sure how to word it properly.